.circleci/config.yml

```diff
@@ -96,12 +96,18 @@ jobs:
       - restore_cache:
           key: dependency-cache-{{ checksum "package-lock.json" }}
       - run:
-          name: Get wiremock # match version in docker-compose-test.yml
-          command: curl -o wiremock.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-jre8-standalone/2.35.0/wiremock-jre8-standalone-2.35.0.jar
+          name: Get wiremock (latest version)
+          command: |
+            LATEST_WIREMOCK_VERSION=$(curl --silent https://repo1.maven.org/maven2/org/wiremock/wiremock-standalone/maven-metadata.xml | sed -n 's/[[:space:]]*<latest>\(.*\)<\/latest>/\1/p')
```

Comment:
Was stumped for quite a while with a mismatch between Wiremock versions locally and in CircleCI - and breaking changes were making tests fail.

Locally, `docker-compose-test.yml` pulls in the image `wiremock/wiremock` which is (currently) `v3.1.0` whereas CircleCI was hardcoded to use version 2.

What I've added here is scripting to download the latest published standalone Wiremock `.jar` version which should - hopefully - match up with the version published to Docker Hub.

Figured this approach was better than pinning both to a specific version 🤷‍♂️